components/dialog/index.css

```diff
@@ -43,24 +43,28 @@
 	/* Allow 100% width, taking into account padding */
 	box-sizing: border-box;
 
-	/* Be no bigger than max-width,  but also be 90% if the viewport is smaller than max-width */
-	inline-size: fit-content;
+	/* Be no bigger than max-width, but also be 90% if the viewport is smaller than max-width */
+	inline-size: var(--mod-dialog-confirm-medium-width, var(--spectrum-dialog-confirm-medium-width));
```

Comment:
I updated this property to match the `.spectrum-Dialog--medium` definition. There are no `--sizeM` specific styles- medium is the default so all of the medium styles are just under `.spectrum-Dialog`.

Does that make sense to do that here with these changes?

Reply:
This is fine since `.spectrum-Dialog--medium` is getting deprecated anyway, but since `.spectrum-Dialog` and `.spectrum--medium` would, in theory, have all the same styles, is there a good reason to keep the `.spectrum--medium` selector in a section by itself? What are your thoughts on moving it up and combining it with the `.spectrum-Dialog` selector? So until it's deprecated, the selector would be

```
.spectrum-Dialog,
.spectrum-Dialog--medium {
```

Reply:
Yes, I like that much better.